### PR TITLE
Revamp the Footer Section

### PR DESCRIPTION
**Pull request by vipulgupta2048**
:hatching_chick: 
- Drop Facebook, added ALiAS Twitter Timeline Embed
- Add several new links, and descriptions. Looks quite better now. 

Solves #97 and partially #145 
Regarding spam on Telegram, let's see if it happens first, before jumping to conclusions. 

Screenshot_2019-10-19_02-46-01

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
Oo also removed the "Contribute to the Website" link from the footer, was eating up space for no good reason.

**Comment by thisisayush** [comment]
Text should be reduced. It'll overflow in mobile devices and look wierd.

File: showcase.html [hunk 1]
```diff
@@ -495,47 +475,50 @@ <h3>Subscribe us on YouTube</h3>
...
+                            <a href="https://asetalias.in/blog" target="_blank">ALiAS Writes, read about the ALiAS Community</a>
                         </li>
                         <li>
                             <i class="fa icon-grey fa-linkedin-square text-blue"></i>
-                            <a href="https://www.linkedin.com/company/asetalias/" target="_blank">LinkedIn</a>
+                            <a href="https://www.linkedin.com/company/asetalias/" target="_blank">LinkedIn, join the largest FOSS community in Amity University</a>
+                        </li>
+                        <li>
+                            <i class="fa icon-grey fa-facebook text-blue"></i>
+                            <a href="https://www.linkedin.com/company/asetalias/" target="_blank">Facebook, join a group of 500+ hackers contributing to FOSS</a>
```
Comment (thisisayush) [inline showcase.html]: The link is incorrect, please check. Also, the one below it for Instagram.

Comment (vipulgupta2048) [reply showcase.html]: Supremely, my bad. Resolved.

**Comment by vipulgupta2048** [comment]
@thisisayush How about this now?

**Comment by thisisayush** [comment]
Looks Good.

**Comment by vipulgupta2048** [comment]
Nope, it's a valid change that I intended to make for optimizing assets and reduce the repository size.

**Comment by thisisayush** [comment]
Yes, please. The mentioned commit can go to Optimize Website Issue #118

**Comment by vipulgupta2048** [comment]
Ready for launch

**Comment by thisisayush** [comment]
Merging.

**Pull request merged**